src/components/SearchJob.jsx

```diff
@@ -3,23 +3,69 @@ import {
 	Form,
 } from 'react-bootstrap';
 import cn from 'classnames';
+import styled from 'styled-components';
 
 function SearchJob() {
+	const FormWrapper = styled.div`
+  @media (max-width: 768px) {
+    position: fixed;
+    z-index: 5;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    transform: translateX(-100%);
+    transition: transform .3s ease-in-out;
+    background-color: rgb(27, 37, 12);
+    &.active {
+      transform: translateX(0);
+    }
+  }
+
+	`;
+	const StyledForm = styled(Form)`
+		margin-top: 15px;
+		border: solid 1px rgb(39, 39, 39);
+		border-radius: 5px;
+		width: 100%;
+		justify-content: center;
+`;
```

Comment:
Indentation is fucked, your eslint is broken

Reply:
i sorted this manually too